src/test/java/com/wikia/webdriver/testcases/adstests/TestFloorAdhesion.java

```diff
@@ -0,0 +1,90 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.dataprovider.mobile.MobileAdsDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.TemplateDontLogout;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.mobile.MobileAdsBaseObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class TestFloorAdhesion extends TemplateDontLogout {
+    private final String FLOOR_ADHESION_SELECTOR = "#ext-wikia-adEngine-template-footer";
```

Comment:
Yes, _CSS_SELECTOR makes sense as well :-)

On 06/01/2015 09:41 PM, Ludwik wrote:
> In
> src/test/java/com/wikia/webdriver/testcases/adstests/TestFloorAdhesion.java
> <https://github.com/Wikia/selenium-tests/pull/661#discussion_r31461894>:
>
>> @@ -0,0 +1,90 @@
>> +package com.wikia.webdriver.testcases.adstests;
>> +
>> +import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
>> +import com.wikia.webdriver.common.dataprovider.mobile.MobileAdsDataProvider;
>> +import com.wikia.webdriver.common.logging.PageObjectLogging;
>> +import com.wikia.webdriver.common.templates.TemplateDontLogout;
>> +import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
>> +import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.mobile.MobileAdsBaseObject;
>> +
>> +import org.openqa.selenium.By;
>> +import org.openqa.selenium.WebElement;
>> +import org.testng.annotations.Test;
>> +
>> +public class TestFloorAdhesion extends TemplateDontLogout {
>> +    private final String FLOOR_ADHESION_SELECTOR = "#ext-wikia-adEngine-template-footer";
>
> so maybe |..._CSS_SELECTOR| :P When writing a test it is easier to see
> your own mistake (or during code reviewing) when using:
>
> |webdriver.findElement(By.cssSelector(MY_UBER_CSS));
> webdriver.findElement(By.xpath(MY_UBER_XPATH));
> |
>
> instead of:
>
> |webdriver.findElement(By.xpath(MY_UBER_SELECTOR));
> webdriver.findElement(By.cssSelector(MY_UBER_SELECTOR));
>
> |
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/selenium-tests/pull/661/files#r31461894>.
>